asb-ballerina/Module.md

```diff
@@ -1,473 +1,262 @@
 
-## Operations on Connecting to the ASB
-The `ballerinax/asb` module contains operations related to connecting with the azure service bus. It includes operations 
-to create SenderConnection, close SenderConnection, create ReceiverConnection, close ReceiverConnection. 
-
-## Operations on Queue
-The `ballerinax/asb` module contains operations on azure service bus Queue. It includes operations to send message to 
-queue, receive message from queue, receive messages from queue, send batch of messages to queue, receive batch of 
-messages from queue, complete messages from queue, complete message from queue, and abandon message from queue.  
-
-## Operations on Topic/Subscription
-The `ballerinax/asb` module contains operations on azure service bus Topic/Subscription. It includes operations to send 
-message to topic, receive message from subscription, receive messages from subscription, send batch of messages to 
-topic, receive batch of messages from subscription, complete messages from subscription, complete message from 
-subscription, and abandon message from subscription.  
-
-## Listener Capabilities
-The `ballerinax/asb` module contains operations related to asynchronous message listening capabilities from the azure 
-service bus. It includes operations to attach service, start listening, detach service, graceful stop listening, and 
-immediate stop listening.
-
-# Prerequisites:
-
-* An Azure account and subscription.
+* Create Azure account and subscription.
   If you don't have an Azure subscription, [sign up for a free Azure account](https://azure.microsoft.com/free/).
 
-* A Service Bus namespace.
+* Create Service Bus namespace.
   If you don't have [a service bus namespace](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-create-namespace-portal),
   learn how to create your Service Bus namespace.
 
-* A messaging entity, such as a queue, topic or subscription.
+* Create messaging entity, such as a queue, topic or subscription.
   If you don't have these items, learn how to
     * [Create a queue in the Azure portal](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#create-a-queue-in-the-azure-portal)
     * [Create a Topic using the Azure portal](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-topics-subscriptions-portal#create-a-topic-using-the-azure-portal)
     * [Create Subscriptions to the Topic](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-topics-subscriptions-portal#create-subscriptions-to-the-topic)
 
-* Java 11 Installed
-  Java Development Kit (JDK) with version 11 is required.
-
-* Ballerina SLAlpha5 Installed
-  Ballerina Swan Lake Alpha 5 is required.
+* Obtain tokens
 
-* Shared Access Signature (SAS) Authentication Credentials
+    Shared Access Signature (SAS) Authentication Credentials are required to communicate with the Azure Service Bus.
```

Comment:
Let's remove unnecessary spaces.

Reply:
Are these sequential steps? If so shall we add numbering? if not shall we add bullet points?

Reply:
This sentence is indented for a proper view. It's not an unnecessary space.

I have added numbering for the sequential steps and bullet points where it's not. Thank you for the suggestion.